hugo/content/guides/formatting.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Formatting"
+weight: 300
+---
+
+Langium's formatting API allows to easily create formatters for your language.
+We start building a custom formatter for our language by creating a new class that inherits from `AbstractFormatter`.
+
+```ts
+import { AbstractFormatter, AstNode, Formatting } from 'langium';
+
+export class CustomModelFormatter extends AbstractFormatter {
+    protected format(node: AstNode): void {
+        // This method is called for every AstNode in a document
+    }
+}
+...
+// Bind the class in your module
+export const CustomModelModule: Module<CustomModelServices, PartialLangiumServices> = {
```

Comment:
Coma in this line is highlighted:
<img width="442" alt="Screenshot 2022-11-23 at 15 34 46" src="https://user-images.githubusercontent.com/15619772/203573303-fda9d38a-e90e-45e8-9641-d1266578f344.png">

Reply:
This is an existing issue with the syntax highlighting. I've seen it in other changes as well. We should address it separately.